projects/Mallard/android/app/src/main/java/com/guardian/editions/ophan/RNOphanModule.java

```diff
@@ -67,10 +66,30 @@ public void setUserId(@Nullable String userId, Promise promise) {
     }
 
     @ReactMethod
-    public void sendTestAppScreenEvent(String screenName, Promise promise) {
+    public void sendAppScreenEvent(@Nonnull String screenName, @Nullable String value, Promise promise) {
+        try {
+            ophanApi.sendAppScreenEvent(screenName, value, UUID.randomUUID().toString());
```

Comment:
Can we assign `UUID.randomUUID().toString()` to a `String` var so we know what it is corresponding to?

`String whatever_this_is = UUID.randomUUID().toString();`

Reply:
This will actually move to the Kotlin layer or further down as this is a bit of an unnecessary implementation. I will add that to the native tidy up ticket.

Updated the code as requested.